src/main/java/uk/gov/ons/ctp/response/action/message/ActionInstructionPublisher.java

```diff
@@ -36,6 +40,11 @@ public void sendActionInstruction(@Header("HANDLER") final String handler, final
       instruction.setActionCancel((ActionCancel) action);
     }
 
+    if (ActionRequestValidator.ACTIONEXPORTER.equalsIgnoreCase(handler)) {
+      actionExportService.acceptInstruction(instruction);
+    }
+
+    // TODO we should be able to remote this soon
```

Comment:
remote?  Do you mean remove?
Also, maybe put more context as to when it can be removed i.e., 'can be removed once we've switched to using pubsub fully'

Reply:
I think this can be removed now